src/patternfly/components/AppLauncher/examples/app-launcher-collapsed-example.hbs

```diff
@@ -1,2 +1,8 @@
-{{#> app-launcher id="app-launcher-example"}}{{/app-launcher}}
+{{#> app-launcher id="app-launcher-example"}}
+  {{#> app-launcher-menu}}
+    <li>{{#> app-launcher-menu-item app-launcher-menu-item--attribute='href="#"'}}Link{{/app-launcher-menu-item}}</li>
```

Comment:
Can we make the menu items in the collapsed and expanded examples match, so they're basically the same example, just in the collapsed and expanded states? I think that's why it was a single include before.